### PR TITLE
Train and deploy end-to-end sklearn text classifier

### PR DESCRIPTION
**Pull request by maxhardt**
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/master/notebooks/official) folder, follow this mandatory checklist:
- [ ] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/notebooks/notebook_template.ipynb) as a starting point.
- [ ] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [ ] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/master/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/docs/CODEOWNERS) file under the `# Community Notebooks` section, pointing to the author or the author's team.

If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/master/community-content) folder:
- [x] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [x] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/docs/CODEOWNERS) file under the `# Community Content` section, pointing to the author or the author's team.

### CODE REVIEW

File: community-content/sklearn_text_classification_from_script_using_vertex_sdk/create_job.ipynb [hunk 1]
```diff
@@ -0,0 +1,286 @@
+{
```
Comment (andrewferlitsch) [inline community-content/sklearn_text_classification_from_script_using_vertex_sdk/create_job.ipynb:1]: <p>The container images appears to be hardwired to "europe". You can derive this by the setting of LOCATION.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (andrewferlitsch) [inline community-content/sklearn_text_classification_from_script_using_vertex_sdk/create_job.ipynb:1]: <p>consider providing a default name for the model display name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (andrewferlitsch) [inline community-content/sklearn_text_classification_from_script_using_vertex_sdk/create_job.ipynb:1]: <p>consider giving a default name to the endpoint display name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

File: community-content/sklearn_text_classification_from_script_using_vertex_sdk/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Train and deploy a sklearn model with Vertex AI
+
+This repository shows how to train and deploy a text classifier using sklearn and Vertex AI.
```
Comment (andrewferlitsch) [inline community-content/sklearn_text_classification_from_script_using_vertex_sdk/README.md:3]: sklearn -> scikit-learn

**Review by andrewferlitsch — APPROVED** [review]

File: community-content/sklearn_text_classification_from_script_using_vertex_sdk/create_job.ipynb (hunk 1 above)
Comment (maxhardt) [reply community-content/sklearn_text_classification_from_script_using_vertex_sdk/create_job.ipynb:1]: Addressed by setting 
```python
LOCATION = f"{REGION}-{ZONE}"`
IMAGE = f"{REGION}-docker.pkg.dev/vertex-ai/training/scikit-learn-cpu.0-23:latest"
```

However, I think the Python client could set this automatically to the closest region, similar to how it's done in the Cloud Console. [From the docs](https://cloud.google.com/vertex-ai/docs/training/pre-built-containers):

> If you use the Google Cloud Console to perform custom training, the Cloud Console selects the URI that best matches the location where you are using Vertex AI in order to reduce latency.

**Comment by kweinmeister** [comment]
Thanks for your contribution @maxhardt!

**Pull request merged**